docs/hpc/03_storage/06_data_management.mdx

```diff
@@ -0,0 +1,115 @@
+
+### HPC Work
+The HPC team makes available a number of public sets that are commonly used in analysis jobs. The data sets are available Read-Only under **`/scratch/work/public`**. 
+
+For some of the datasets users must provide a signed usage agreement before accessing.
+
+Public datasets available on the HPC clusters can be viewed on the [Datasets page](../01_getting_started/01_intro.md).
+
+### HPC Archive
+Once the Analysis stage of the research data lifecycle has completed, <ins>_HPC users should **tar** their data and code into a single tar.gz file and then copy the  file to their archive directory (**`/archive/$USER`**_).</ins> The HPC Archive file system is not accessible by running jobs; it is suitable for long-term data storage. Each user has access to a default disk quota of **2TB** and ***20,000 inode (files) limit***. The rather low limit on the number of inodes per user is intentional.  The archive file system is available only ***on login nodes*** of Greene. The archive file system is backed up daily.
+
+-   Here is an example ***tar*** command that combines the data in a directory named ***my_run_dir*** under ***`$SCRATCH`*** and outputs the tar file in the user's ***`$ARCHIVE`***: 
+```sh
+# to archive `$SCRATCH/my_run_dir`
+tar cvf $ARCHIVE/simulation_01.tar -C $SCRATCH my_run_dir
+```
+
+### NYU (Google) Drive
+Google Drive ([NYU Drive](https://www.nyu.edu/life/information-technology/communication-and-collaboration/document-collaboration-and-sharing/nyu-drive.html)) is accessible from the NYU HPC environment and provides an option to users who wish to archive data or share data with external collaborators who do not have access to the NYU HPC environment. 
+
+Currently (March 2021) there is no limit on the amount of data a user can store on Google Drive and there is no cost associated with storing data on Google Drive (although we hear rumors that free storage on Google Drive may be ending soon).
+
+However, there are limits to the data transfer rate in moving to/from Google Drive. Thus, moving many small files to Google Drive is not going to be efficient. 
+
+Please read the [Instructions on how to use cloud storage within the NYU HPC Environment](./12_transferring_cloud_storage_data_with_rclone.md). 
+
+### HPC Storage Mounts Comparison Table
+<iframe width="100%" height="300em" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vT-q0rRueYg1Be_gcWSghB-GGFDonP8DaXNnm8Qi036w-Vi_l7CCOav4IPxi1yZy8TSnTRFy7S5dNTJ/pubhtml?widget=true&amp;headers=false"></iframe>
+
+## Data Management Subpages
+### Data Transfers 
+[Link to Data Transfers Page](./03_data_transfers.md)
+
+### Sharing Data on HPC 
+[Link to Sharing Data on HPC Page](./13_sharing_data_on_hpc.md)
+
+### Research Project Space 
+[Link to Research Project Space Page](./05_research_project_space.md)
+
+### Squash File System and Singularity
+[Link to Squash File System and Singularity Page](../07_containers/04_squash_file_system_and_singularity.md)
```

Comment:
Remove the section of links as it doesn't add anything that's not already in the text. Maybe add a line about being able to learn about best practices for data management in the next section?

Reply:
Yeah, you're right, this is basically the same as the navigation we had at the tops of pages.  I've removed it and added the line you suggested. Thanks!